### PR TITLE
Blog post on graduate communities

### PR DESCRIPTION
**Pull request by DamienIrving**
@gvwilson Hope you don't mind me dropping your name in this post...

### CODE REVIEW

File: blog/2015/10/graduate_communities.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  and in other cases we could approach people in the Software Carpentry community to help establish them 
+  (e.g. Software Carpentry graduates who do bioinformatics using R). 
+  These official Software Carpentry alumni groups would have a mailing list for general chat and questions 
+  (that people are automatically signed up to after attending a workshop), 
+  a blog, a discipline and language specific capstone lesson that they maintain
+  and they may even host workshops at relevant conferences 
+  (e.g. PyAOS runs a workshop at the American Meteorological Society Annual Meeting every year)."  
+</p>
+<p>
+  What do people think? Is this an initiative that the Steering Committee should look at implementing?
```
Comment (rgaiacs) [inline blog/2015/10/graduate_communities.html]: > Is this an initiative that the Steering Committee should look at implementing?

This was one of the reasons that we helped NumFOCUS with this year edition of GSoC.

File: blog/2015/10/graduate_communities.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+---
+<!-- start excerpt -->
+<p>
+  I was chatting to Greg Wilson via email the other day,
+  when I got sidetracked on a long-winded rant about the big issues facing Software Carpentry.
+  (I might be one of the reasons Greg has trouble clearing his inbox...) 
+  I thought I'd share an excerpt from my email here, 
+  as a possible solution to the issue of supporting learners 
+  after they've attended one of our workshops:
+</p>
```
Comment (gvwilson) [inline blog/2015/10/graduate_communities.html]: I would drop the entire opening paragraph (see below).

File: blog/2015/10/graduate_communities.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+<!-- end excerpt -->
+<p>
+  "Besides the instructor training bottleneck, 
+  it seems to me that the other huge issue facing Software Carpentry
+  is what happens to learners after they've attended a workshop.
```
Comment (gvwilson) [inline blog/2015/10/graduate_communities.html]: Use this as the opening paragraph, and cut the lead to, "One of the issues I'd like Software Carpentry to tackle is what happens to learners <em>after</em> they've attended a workshop."

File: blog/2015/10/graduate_communities.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+  as a possible solution to the issue of supporting learners 
+  after they've attended one of our workshops:
+</p>
+<!-- end excerpt -->
+<p>
+  "Besides the instructor training bottleneck, 
+  it seems to me that the other huge issue facing Software Carpentry
+  is what happens to learners after they've attended a workshop. 
+  Thinking about my own experience, 
+  I attended the workshop you ran in Melbourne in early 2013
```
Comment (gvwilson) [inline blog/2015/10/graduate_communities.html]: cut "you ran"

File: blog/2015/10/graduate_communities.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+</p>
+<!-- end excerpt -->
+<p>
+  "Besides the instructor training bottleneck, 
+  it seems to me that the other huge issue facing Software Carpentry
+  is what happens to learners after they've attended a workshop. 
+  Thinking about my own experience, 
+  I attended the workshop you ran in Melbourne in early 2013 
+  and then 2.5 years later I published my first ever
+  <a href="https://www.authorea.com/users/5641/articles/12197/_show_article">reproducible paper</a>
```
Comment (gvwilson) [inline blog/2015/10/graduate_communities.html]: Full stop at the end of this line.

File: blog/2015/10/graduate_communities.html [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  (as I continually have to explain to impatient managers at Research Platform Services,
```
Comment (gvwilson) [inline blog/2015/10/graduate_communities.html]: Title-cas "As" and move the full stop on the next line inside the paren.

File: blog/2015/10/graduate_communities.html [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+  <a href="https://www.authorea.com/users/5641/articles/12197/_show_article">reproducible paper</a>
+  (as I continually have to explain to impatient managers at Research Platform Services, 
+  yes it will take people at least that long to fully work these things into their daily workflow). 
+  The difference between me 
+  and a Software Carpentry attendee who never goes on to adopt the best practices we teach 
+  is probably not that I read better books, attend better follow-up classes or find better online tutorials, 
+  but rather that I'm part of an active community. 
+  In fact, besides that initial Software Carpentry workshop,
+  I can't recall a single book, class or tutorial that was hugely influential in my development as a computational scientist. 
+  Your two-day Software Carpentry workshop in 2013 introduced me to the core concepts I needed to know,
```
Comment (gvwilson) [inline blog/2015/10/graduate_communities.html]: Change "your" to "the"

**Comment by DamienIrving** [comment]
Thanks, @gvwilson. I was being lazy with the straight cut and paste from my email and have made the changes you suggested.

**Pull request merged**